RockWeb/web.config

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Do we really need changes to the `web.config` for this to work? Or are/can the redirects be handled someplace else? I don't mind that this is here if we need it. But we will have to get Paul to change the actual `web.config` on the servers too. This is because during the deploy we move the `web.config` file to another place for safe keeping (so Rock doesn't overwrite ours) and then we put it back when we're done.

Reply:
This was added because we added the http > https url rewrite in IIS. This fixes the issue where on our non-appveyor environments (dev instances) would not resolve unless you explicitly call for https. Appveyor backs up the web.config file from alpha, beta, and prod and restores it during the deploy, so this file under version control is just for development, according to Paul.

Reply:
Yep, that's true. He's been trying to keep the servers version fairly up-to-date with what we have in the repo, but if he doesn't need this part, then we're good. 💯